packages/nextjs/hooks/useFetchUserData.tsx

```diff
@@ -40,6 +41,7 @@ export const useFetchUserData = ({ address }: { address?: string }) => {
     refetchInterval: 10000,
   });
 
+  // challenge1 only exists in season 1
```

Comment:
do we need to change something here?

Reply:
No, the comment is because when finding the challenge 1, the season is not taken into account.